printer/testdata/raw_string.expect

```diff
@@ -0,0 +1,11 @@
+block "label" {
```

Comment:
labels get printed out with double quotes instead of the original backticks. I think that's probably fine for labels but I wanted to call attention to it.